### ISSUE TITLE
Not able to load package after installation

### ISSUE THREAD

**Issue by massimoaria** [OscarKjell/text#199]
I installed text, but when I try to load it R console returns this error:

library(text)
This is text (version 1.2.3).
Text is new and still rapidly improving.
               
Newer versions may have improved functions and updated defaults to reflect current understandings of the state-of-the-art.
               Please send us feedback based on your experience.

For more information about the package see www.r-text.org.
textrpp python option is already set, text will use: condaenv = "textrpp_condaenv"
Error: package or namespace load failed for ‘text’:
 .onAttach failed in attachNamespace() for 'text', details:
  call: reticulate::use_condaenv(settings$val, required = TRUE)
  error: Unable to locate conda environment 'textrpp_condaenv'.

**Comment by OscarKjell** [comment]
have you tried running: 
textrpp_install()
textrpp_initialize()

**Comment by massimoaria** [comment]
Yes, but the console returns an error message "function doesn't exist" because I can't load the library text. 
Here the console output:

`library(text)
This is text (version 1.2.3).
Text is new and still rapidly improving.
               
Newer versions may have improved functions and updated defaults to reflect current understandings of the state-of-the-art.
               Please send us feedback based on your experience.

For more information about the package see www.r-text.org.
textrpp python option is already set, text will use: condaenv = "textrpp_condaenv"
Error: package or namespace load failed for ‘text’:
 .onAttach failed in attachNamespace() for 'text', details:
  call: reticulate::use_condaenv(settings$val, required = TRUE)
  error: Unable to locate conda environment 'textrpp_condaenv'.
>   textrpp_install()
Error in textrpp_install() : could not find function "textrpp_install"
>   textrpp_initialize()
Error in textrpp_initialize() : 
  could not find function "textrpp_initialize"`

**Comment by OscarKjell** [comment]
oh, i have not seen this error before. Could you please try to install the development version from github? 

devtools::[install_github](https://remotes.r-lib.org/reference/install_github.html)("oscarkjell/text")

**Comment by lingdoc** [comment]
seems this is the same error generated by #221 - essentially the `text` installer thinks there is already a conda env called `textrpp_condaenv` and errors out when it can't find it. default behavior (imo) should be to overwrite it and create a new env if the text library is not installed.
